client/src/components/aboutUs.jsx

```diff
@@ -19,19 +19,18 @@ function AboutUs() {
             complex course structures and deciphering prerequisites. That's why we've 
             developed a platform that allows students to directly see dependent courses, 
             easily access related courses, and presents prerequisites in a clear format.
-        </p>
+          </p>
         </div>
-        <img className='image1'
+        <img className='relatableFoundersImages'
           src="../images/about_us_image1.jpg"
         ></img>
       </div>
 
-
-      <div className='sections'>
-        <img className='image1'
+      <div className='aboutUsRowContainer'>
+        <img className='relatableFoundersImages'
           src="../images/about_us_image2.jpg"
```

Comment:
Can we SEO-friendlier names for the images like "Relatable Student Group"? Also, as I noted previously, let's at alt text to these img tags